app/assets/javascripts/stores/message_store.js.coffee

```diff
@@ -0,0 +1,55 @@
+window.messageStore = Reflux.createStore
+  init: ->
+    @listenToMany(userActions)
+    @listenTo remoteActions.message, 'onReceiveMessage'
+    @state = @getInitialState()
```

Comment:
You're right.. I've seen some people do it this way:
```coffee
initialState = { ... }
window.messageStore = Reflux.createStore
  init: ->
    @state = @initial
  getInitialState: ->
    @initial
```

Could be a question to ask in the Reflux channel of the Reactiflux slack.